src/VirtoCommerce.QuoteModule.ExperienceApi/Queries/QuotesQuery.cs

```diff
@@ -37,5 +39,6 @@ public override void Map(IResolveFieldContext context)
         CurrencyCode = context.GetArgument<string>(nameof(CurrencyCode));
         CultureName = context.GetArgument<string>(nameof(CultureName));
         Filter = context.GetArgument<string>(nameof(Filter));
+        OrganizationId = context.GetArgument<string>(nameof(OrganizationId));
```

Comment:
Need to check if current user belongs to provided organization.
Or just use curent organization.